tests/cpp/test_simplex.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright 2025 D-Wave Inc.
```

Comment:
```suggestion
// Copyright 2025 D-Wave
```
Nice job on making all three headers different! :laughing:

Reply:
Gotta keep reviewers on their toes.